wiki/People/Community_Contributors/de.md

```diff
@@ -1,33 +1,220 @@
+
+| Benutzer | Beiträge |
+| :-- | :-- |
+| ![][flag_DE] [OnosakiHito](https://osu.ppy.sh/users/290128) | Herausragende Arbeit bei der Gründung der frühen osu!taiko Community, mehrjährige Tätigkeit im BAT/QAT |
+| ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565) | Herausragende Beiträge zur osu!catch Community, Überarbeitung der Ranking-Kriterien und Wettbewerben |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | Herausragende Beiträge zu zahlreichen Medienprojekten, Wettbewerben und eine langjährige Zugehörigkeit zum GMT |
+| ![][flag_US] [Garven](https://osu.ppy.sh/users/244216) | Jahrelange engagierte Arbeit für das BVT/QAT und immenser Beitrag zur Überarbeitung der Ranking-Kriterien |
+| ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | Jahrelanges Engagement für das BAT/QAT, maßgebliche Beteiligung an der Überarbeitung der Ranking-Kriterien und am Beatmap Nominator Test Management |
+| ![][flag_CH] [Irreversible](https://osu.ppy.sh/users/1287964) | Engagierter Beitrag zum BAT/QAT über unzählige Jahre hinweg |
+| ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) | Hunderte von Stunden an Statistiken und Analysen zu den Wettbewerben des World Cup |
+| ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) | Herausragende Präsenz als Schiedsrichter bei Dutzenden von Community Wettbewerben |
+| ![][flag_CA] [Evrien](https://osu.ppy.sh/users/791660) | Hervorragende Leistungen bei Castings, Kommentieren und ereignisbezogenen Berichten/Recaps |
+| ![][flag_DE] [Tom94](https://osu.ppy.sh/users/1857058) | Der Kopf hinter zahllosen Verbesserungen von osu!, von pp, einer neu geschriebenen Grafik-Engine, Star-Rating und mehr! |
+| ![][flag_CA] [DrabWeb](https://osu.ppy.sh/users/6946022) | Herausragender Beitrag zum osu!lazer-Projekt |
+| ![][flag_BY] [EVAST](https://osu.ppy.sh/users/8195163) | Herausragender Beitrag zum osu!lazer-Projekt mit über 90 Pullreqs und Hunderten von Commits |
+| ![][flag_CN] [huoyaoyuan](https://osu.ppy.sh/users/2428732) | Herausragender Beitrag zum osu!lazer-Projekt |
+| ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) | Herausragender Beitrag und Organisation des chinesischen Lokalisierungsprojekts für osu!, osu! wiki und osu!lazer |
+| ![][flag_DE] [jorolf](https://osu.ppy.sh/users/7004641) | Herausragender Beitrag zum osu!lazer-Projekt und Erstellung zahlreicher Medienwerkzeuge |
+| ![][flag_AU] [Syrin](https://osu.ppy.sh/users/5701575) | Ersteller von PerformancePlus und osu!chan |
+| ![][flag_SG] [Raveille](https://osu.ppy.sh/users/1388767) | Herausragende Leistungen bei der Erstellung und Veröffentlichung des Scorewatch-Projekts |
+| ![][flag_FR] [ThePooN](https://osu.ppy.sh/users/718454) | Herausragende Leistungen bei der Erstellung und Veröffentlichung des Scorewatch-Projekts |
+| ![][flag_US] [MegaApple_Pi](https://osu.ppy.sh/users/2148208) | Hervorragender Einsatz bei der Förderung des osu! Wiki-Projekts mit unzähligen Überarbeitungen, Nacharbeiten und Reviews |
```

Comment:
```suggestion
| ![][flag_US] [MegaApple_Pi](https://osu.ppy.sh/users/2148208) | Hervorragender Einsatz bei der Förderung des osu!-Wiki-Projekts mit unzähligen Überarbeitungen, Nacharbeiten und Überprüfungen |
```
`Bewertung` instead of `Überprüfungen` is also fine to me, in case you want to take this.

Reply:
`Überprüfungen` sounds good.